muda/deformers/filter.py

```diff
@@ -0,0 +1,538 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+# CREATED:2020-07-11 by Han Han <hh2263@nyu.edu>
+"""Filtering (low/band/high-pass) algorithms"""
+
+import numpy as np
+from scipy import signal
+from copy import deepcopy
+import numpy as np
+import librosa
+
+from ..base import BaseTransformer, _get_rng
+
+__all__ = ["Filter", "RandomLPFilter", "RandomHPFilter","RandomBPFilter"]
+
+
+def checkfreqinband(freq,state,datatype):
```

Comment:
Can you add a docstring for this function, explaining what its inputs should be and what it returns?  See _get_rng for an example of how that sort of thing should look.